Subject: Fixes to systemd service files
From: Dominik George <nik@naturalnet.de>
Forwarded: https://github.com/neutrinolabs/xrdp/pull/646
--- a/instfiles/xrdp-sesman.service.in	2017-08-04 22:45:35.434283541 +0200
+++ b/instfiles/xrdp-sesman.service.in	2017-08-04 22:46:43.682007170 +0200
@@ -7,7 +7,8 @@
 
 [Service]
 Type=forking
-PIDFile=@localstatedir@/run/xrdp-sesman.pid
+PIDFile=/run/xrdp/xrdp-sesman.pid
+RuntimeDirectory=xrdp
 EnvironmentFile=-@sysconfdir@/sysconfig/xrdp
 EnvironmentFile=-@sysconfdir@/default/xrdp
 ExecStart=@sbindir@/xrdp-sesman $SESMAN_OPTIONS
--- a/instfiles/xrdp.service.in	2017-08-04 22:45:35.434283541 +0200
+++ b/instfiles/xrdp.service.in	2017-08-04 22:48:21.853609617 +0200
@@ -6,9 +6,12 @@
 
 [Service]
 Type=forking
-PIDFile=@localstatedir@/run/xrdp.pid
+PIDFile=/run/xrdp/xrdp.pid
+RuntimeDirectory=xrdp
 EnvironmentFile=-@sysconfdir@/sysconfig/xrdp
 EnvironmentFile=-@sysconfdir@/default/xrdp
+User=xrdp
+Group=xrdp
 PermissionsStartOnly=true
 ExecStartPre=/bin/sh /usr/share/xrdp/socksetup
 ExecStart=@sbindir@/xrdp $XRDP_OPTIONS
